### ISSUE TITLE
Liking (stars) for maps

### ISSUE THREAD

**Issue by jywarren** [publiclab/mapknitter#126]
just like on publiclab.org

**Comment by kaunair** [comment]
Hey! I'd like to take this issue!
I'm thinking of creating a model called map_likes with map_id and user_id as attributes. Would this be sufficient or should I rename it to something else in case you would incorporate other features like followers?
Also should there be any email notification sent after liking?

**Comment by jywarren** [comment]
Let's name the model `map_reactions` and have a `reaction_type` column that
has default `like`, but we could add `follow` or others in the future. How
does that sound? Thanks, and we'd love help -- i do think we are /very/
interested in full unit testing for any new feature so please bear that in
mind!

**Comment by kaunair** [comment]
Sound's good. What other kind of reactions are we having? Github like reactions?

**Comment by jywarren** [comment]
potentially -- best to just plan ahead!

But wait - what if we thought a bit more about this and made reactions to
individual images instead of to maps? We are considering making the
platform more image centric in any case. What do you think?

**Comment by kaunair** [comment]
Interesting..
I'm new to Mapknitter here, isn't the point to stitch the images together to create a map?
What's more important to the user: the individual images or the stitched map?
But if we're going to be more image centric then adding reactions to images makes more sense!

**Comment by jywarren** [comment]
Yes, i think though that there's a conceptual change we're considering,
especially because it may be faster to just start uploading images instead
of starting by "creating a map", and then people can just select any group
of images to generate an export. Then images can "belong" to multiple
collections for export. We're not 100% set on it but it does seem like a
simpler mental model with a few less steps. I think either way, it may be
useful to "like" or react to individual images rather than whole maps. Is
that helpful? Thanks!

**Comment by kaunair** [comment]
Sounds cool!
Thanks for the help. I'll send a pr soon
